crates/nu-plugin/src/plugin/context.rs

```diff
@@ -150,6 +152,14 @@ impl<'a> PluginExecutionContext for PluginExecutionCommandContext<'a> {
         .into_spanned(self.call.head))
     }
 
+    fn get_span_contents(&self, span: Span) -> Result<Spanned<Vec<u8>>, ShellError> {
+        Ok(self
+            .engine_state
+            .get_span_contents(span)
+            .to_vec()
+            .into_spanned(self.call.head))
```

Comment:
`into_spanned(self.call.head))` seems wrong to me, because we need to get content from given `span`.

So I think this should be `into_spanned(span)`

Reply:
Yeah, I wasn't really sure which to choose... But if the value were going to be returned from the command, it would be better for it to be spanned from the command than from the span of the source code.

It doesn't matter so much for Rust because this span is actually not used in the API. I just didn't want to keep creating unspanned variants of `EngineCallResponse` when there are perfectly usable `Value` types.

Anyway, the reasoning is that the span refers to the source code, but the source code string originates from the command that produced it